### PR TITLE
Added PPG + LSL streamers for each data type - partially fix #79, see also #71 - 

### LINKED ISSUE #79: Add PPG support for Muse 2 devices
The Muse 2 ships with two additional PPG (photoplethysmography) sensors which can be used to obtain measures of heart rhythm. These sensors broadcast their readings through gatt characteristics in much the same way that the 2016 Muse's gyroscope and accelerometer do.

In order to be able to use PPG with this library, we will have to discover the gatt characteristics of the PPG sensors and include handling functionality to make a PPG stream available for subscription. Code that performs this task for the accelerometer and gyroscope sensors of the Muse 2016 is available in this impending PR: https://github.com/alexandrebarachant/muse-lsl/pull/34. Once a Muse 2 is available for testing, it will likely be most efficient for inclusion of the acc and gyro code to occur alongside development of the new PPG handling code.

### PR DESCRIPTION
**Pull request by Remy17**
Hello, thank you very much for your code that was very useful for our projects this year !
We bought the Muse 2, and I think that we found how to add PPG management. Hope it helps !

In constants file : added uuid for PPG
In muse.py file : added PPG management
In stream.py : added LSL streamers for each data type (optional)
In view.py : changed a few things to later add each datatype visualization

### CODE REVIEW

**Comment by Dutchflag** [comment]
Hi @Remy17,

I have tested your code and the ppg results confuse me. 

Did you confirm that PPG has a sample rate of 256 hz, i.e. the same as EEG)?
When I try to handle both EEG and PPG at the same time, I get lots of dropped samples and a very strange interaction between the EEG and PPG data packets...

perhaps I missed something, I'll try additional testing later this week

**Comment by bshapiro6** [comment]
Hi @Dutchflag @Remy17,

Thanks for working on this code! The PPG sampling rate is actually 64 Hz, though.

Ben (from Interaxon)

File: muselsl/viewer_eeg_v1.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
+import numpy as np
```
Comment (jdpigeon) [inline muselsl/viewer_eeg_v1.py]: I think since we'll focus on making the same viewer work for all data types renaming these viewer files is unnecessary.

**Comment by jdpigeon** [comment]
Now deprecated by https://github.com/alexandrebarachant/muse-lsl/pull/93. Thanks for you help!!

**Pull request closed without merging**